README.md

```diff
@@ -62,8 +59,9 @@ module.exports = {
             loader: MiniCssExtractPlugin.loader,
             options: {
               // you can specify a publicPath here
-              // by default it use publicPath in webpackOptions.output
-              publicPath: '../'
+              // by default it uses publicPath in webpackOptions.output
+              publicPath: '../',
+              hot: process.env.NODE_ENV === 'development'
```

Comment:
Let's rename `hot` to `hmr` (we use same name for `style-loader` and we rename `hot` to `hmr` in `webpack-dev-server` in next major release)

Reply:
okay